<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>save_window</class>
 <widget class="QDialog" name="save_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Analysis</string>
  </property>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>181</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>batch-config</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>kind of processing:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_img">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Image:</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>128</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="rdbtn_single">
        <property name="text">
         <string>single</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdbtn_batch">
        <property name="text">
         <string>batch</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QComboBox" name="comboBox_img_list">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkB_thresh_analysis">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Treshold Analysis</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>results</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>31</y>
       <width>361</width>
       <height>76</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="chkB_result_exg">
          <property name="text">
           <string>ExG</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkB_result_exr">
          <property name="text">
           <string>ExR</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QCheckBox" name="chkB_result_exgr">
          <property name="text">
           <string>ExGR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkB_result_filter">
          <property name="text">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkB_auto_filter">
          <property name="text">
           <string>auto filter</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>18</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QCheckBox" name="chkB_result_histogram">
          <property name="text">
           <string>Histogram</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkB_mean_stddev">
          <property name="text">
           <string>Mean &amp; StdDev</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="empty">
          <property name="text">
           <string>(Plant / Soil)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>choose results of the methods to be saved in csv-file:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_file_name_csv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Dataset Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputtxt_file_name_csv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>271</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>images</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>choose images to save:</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>31</y>
       <width>361</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="chkB_img_exg_binary">
          <property name="text">
           <string>ExG Binary</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkB_img_exg_color">
          <property name="text">
           <string>ExG Color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkB_img_exr_binary">
          <property name="text">
           <string>ExR Binary</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkB_img_exr_color">
          <property name="text">
           <string>ExR Color</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="chkB_img_exgr_binary">
          <property name="text">
           <string>ExGR Binary</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkB_img_exgr_color">
          <property name="text">
           <string>ExGR Color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkB_img_filter_binary">
          <property name="text">
           <string>Filter Binary</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkB_img_filter_color">
          <property name="text">
           <string>Filter Color</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>18</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="chkB_img_histogram">
          <property name="text">
           <string>Histogram</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkB_img_EditedImage">
          <property name="text">
           <string>edited Image</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkB_img_all_grey">
          <property name="text">
           <string>all Grey</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="format">
    <string>%p%</string>
   </property>
  </widget>
  <zorder>progressBar</zorder>
  <zorder>btn_start</zorder>
  <zorder>btn_cancel</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rdbtn_single</sender>
   <signal>clicked()</signal>
   <receiver>lbl_img</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdbtn_batch</sender>
   <signal>clicked()</signal>
   <receiver>lbl_img</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdbtn_batch</sender>
   <signal>clicked()</signal>
   <receiver>comboBox_img_list</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdbtn_single</sender>
   <signal>clicked()</signal>
   <receiver>comboBox_img_list</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
